src/components/Link/Link-v2.tsx

```diff
@@ -0,0 +1,96 @@
+import clsx from 'clsx';
+import React, { forwardRef } from 'react';
+import type { Size } from '../../util/variant-types';
+import type { IconName } from '../Icon';
+import Icon from '../Icon';
+
+import styles from './Link-v2.module.css';
+
+type LinkHTMLElementProps = Omit<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  'disabled'
+>;
+
+export type LinkProps = LinkHTMLElementProps & {
+  // Component API
+  /**
+   * Component used to render the element. Meant to support interaction with framework navigation libraries.
+   *
+   * **Default is `"a"`**.
+   */
+  as: string | React.ElementType;
+  /**
+   * The link contents or label.
+   */
+  children: string;
+  // Design API
+  /**
+   * Where `Link` sits alongside other text and content:
+   *
+   * * **inline** - Inline link inherits the text size established within the `<p>` paragraph they are embedded in.
+   * * **standalone** - Users can choose from the available sizes.
+   */
+  context?: 'inline' | 'standalone';
+  /**
+   * (trailing) icon to use with the link
+   */
+  icon?: Extract<IconName, 'chevron-right' | 'open-in-new'>;
+  /**
+   * Extra or lowered colors added to a link
+   */
+  emphasis?: 'default' | 'high' | 'low';
+
+  /**
+   * Link size inherits from the surrounding text.
+   */
+  size?: Extract<Size, 'xs' | 'sm' | 'md' | 'lg' | 'xl'>;
+};
+
+/**
+ * `import {Link} from "@chanzuckerberg/eds";`
+ *
+ * Component for making styled anchor tags. Links allow users to navigate within or between a web page(s) or app(s).
+ *
+ */
+export const Link = forwardRef<HTMLAnchorElement, LinkProps>(
```

Comment:
Can't wait until React removes `forwardRef` in React 19

Reply:
Ohh, what will happen instead? I never read the upcoming docs for things besides HTML/CSS/JS :P